app/src/main/java/agency/five/tmdb/ui/components/MovieCard.kt

```diff
@@ -10,23 +10,32 @@ import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.Painter
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.tooling.preview.Preview
+import coil.compose.rememberImagePainter
+import kotlinx.serialization.Serializable
 
-
+@Serializable
```

Comment:
Check comment from the previous PR about models separation.

We can discuss about that on online consultation, feel free to ping your mentor. 👍